docs/source/examples/private_utxo/private_utxo_transaction_family.rst

```diff
@@ -0,0 +1,411 @@
+***************************************
+Private UTXO Transaction Family
+***************************************
+
+Overview
+========
+
+This document describes a Transaction Family that allow for assets to be
+tracked and traded on the Ledger as well as to be held off the Ledger and still
+traded in a manner that enforces the Ledger rules governing the Asset and
```

Comment:
This sentence has a lot going on. It could do with being split up into multiple sentences.